themes/godotengine/layouts/features.htm

```diff
@@ -100,13 +100,13 @@ <h2>One of the best looking 3D engines out there</h2>
 
   <div class="title-padded base-padding">
     <h2>Create 2D games with ease</h2>
-    <em> Godot comes with a fully dedicated 2D engine packed with a ton of features. </em>
+    <em> Godot comes with a fully dedicated 2D engine packed with helpful features. </em>
```

Comment:
I'm not sure if "helpful" is an improvement here, non-helpful features would be bugs :P